src/widgets/BarChart.js

```diff
@@ -33,7 +33,7 @@ export const BarChart = ({ data, width, height }) => {
 
   return (
     <VictoryChart width={width} height={height} padding={padding} domainPadding={domainPadding}>
-      <VictoryBar style={style} data={values} />
+      <VictoryBar style={style} data={values[0]} />
```

Comment:
This will throw an error if `values.length == 0` 😢 . A safer alternative is to declare a variable by deconstructing:

```
const [value = null] = values;

return (
  ...
     <VictoryBar style={style} data={value} />
  ...
)
```

Using this syntax, `value` will default to `null` if `values.length == 0`.

Reply:
Thanks Will! I was making more changes related with this PR that can be found here:
f5067e2
dba8a2c 
e46724a

I found that data structure can include more than one array of values (mainly on line charts), so I added a map over data to render as many i.e. lines as necessary in the same graph.